lib/resty/upstream/healthcheck.lua

```diff
@@ -524,28 +533,33 @@ end
 
 function _M.spawn_checker(opts)
     local typ = opts.type
-    if not typ then
+    if typ == nil then
         return nil, "\"type\" option required"
     end
 
-    if typ ~= "http" then
-        return nil, "only \"http\" type is supported right now"
+    local host = opts.host
+    if host == nil then
+        host = nil
+    end
+
+    local ssl_verify = opts.ssl_verify
+    if ssl_verify == nil then
+        ssl_verify = true
     end
 
     local http_req = opts.http_req
-    if not http_req then
+    if http_req == nil then
```

Comment:
why need this change?

Reply:
Aha, same purpose as before, but in another way I like.
Anyway, I will revert those lines.